src/vscode-dts/vscode.proposed.interactive.d.ts

```diff
@@ -138,7 +138,22 @@ declare module 'vscode' {
 		treeData: FileTreeData;
 	}
 
-	export type InteractiveProgress = InteractiveProgressContent | InteractiveProgressId | InteractiveProgressTask | InteractiveProgressFileTree;
+	export interface DocumentContext {
```

Comment:
Does this work if there are multiple codeblocks? Does the context need to be associated with one of the codeblocks somehow?

Reply:
We intend for this context to be associated with all codeblocks that occur a single reply for now. We can try associating a code block in the reply with ranges in documents but it's farther in the future.